modules/tinymce/src/core/test/ts/browser/keyboard/SpaceKeyTest.ts

```diff
@@ -16,12 +16,21 @@ describe('browser.tinymce.core.keyboard.SpaceKeyTest', () => {
 
   context('Space key around inline boundary elements', () => {
     it('Press space at beginning of inline boundary inserting nbsp', () => {
+      const inputEvents: { inputType: string; data: string }[] = [];
```

Comment:
```suggestion
      const inputEvents: Array<{ inputType: string; data: string }> = [];
```
Unfortunately there isn't a similar rule to our old TSLint rule here, but complex types should use `Array<>`.

Reply:
Right, I knew about this one. Been out of the TS code so long that I forgot about that rule. :)